changelog.d/628-combinatorial-coverage.md

```
@@ -0,0 +1,3 @@
+### Added
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)